Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -1183,7 +1183,7 @@ private extension ChatMessage {
         $_author = ({ user }, nil)
         $_attachments = ({
             dto.attachments
-                .map { $0.asAnyModel() }
+                .compactMap { $0.asAnyModel() }
```

Comment:
This compactMap here will completely absorb the case where attachments can't be converted. Shall we surface the error before doing the compactMap?

Reply:
There's no reason for it to fail unless there is a logout process going on or something. The logging will make the code a bit harder to read, so I'm not sure it is worth it 🤔

Reply:
It will make it easier to debug and will provide integrators with some logs to communicate with us. Readability won't be affected as the one liner is actually dangerous on hide the error here.

Reply:
Done 👍